pysatMadrigal/instruments/methods/madrigal.py

```diff
@@ -437,17 +426,6 @@ def get_remote_filenames(inst_code=None, kindat=None, user=None,
     else:
         kindat = [int(kk) for kk in kindat.split(",")]
 
-    # currently passes things along if no user and password supplied
-    # need to do this for testing
-    # TODO, implement user and password values in test code
-    # specific to each instrument
-    if user is None:
-        logger.warning('No user information supplied for download.')
-        user = 'pysat_testing'
-    if password is None:
-        logger.warning('Please provide email address in password field.')
-        password = 'pysat_testing@not_real_email.org'
-
```

Comment:
So, warnings should be raised if a user name and email aren't supplied.  They should probably be ValueErrors, though.

Reply:
Updated.  I've gone ahead and just made this a check function that checks for inst_code, user, and password to reduce duplication.